build-logic/binary-compatibility/src/main/groovy/gradlebuild/binarycompatibility/transforms/ExplodeZipAndFindJars.groovy

```diff
@@ -26,12 +26,14 @@ import org.gradle.api.file.FileSystemLocation
 import org.gradle.api.provider.Provider
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
+import org.gradle.work.DisableCachingByDefault
 
 import java.nio.file.Files
 import java.util.zip.ZipEntry
 import java.util.zip.ZipInputStream
 
 @CompileStatic
+@DisableCachingByDefault(because = "Only filters the input artifact")
```

Comment:
Does it? I thought this actually extracts stuff.

Reply:
You are right. Changed to `Not worth caching`.